bucket/webex.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "41.12.0.20899",
+    "description": "Video and audio conferencing",
+    "homepage": "https://www.webex.com/",
+    "license": {
+        "identifier": "Proprietary"
+    },
```

Comment:
Add a URL field too, to their License/Privacy Policy/EULA.

Reply:
Initially I couldn't find one for the application online, but I managed to dig a little deeper and get the EULA.